$OpenBSD: patch-lib_icons_bomb_32x32_xpm,v 1.1 2002/01/20 19:54:13 naddy Exp $
# Fix broken icon, so it can be displayed correctly on Sparc too
--- lib/icons/bomb_32x32.xpm.orig	Sun Jan 13 20:33:42 2002
+++ lib/icons/bomb_32x32.xpm	Sun Jan 13 20:33:56 2002
@@ -1,18 +1,15 @@
 /* XPM */
-static char * icl8_11432_xpm[] = {
-/* width height ncolors chars_per_pixel */
+static char * bomb_32x32_xpm[] = {
 "32 32 9 1",
-/* colors */
-" 	s None	c None",
-"o	c #999999",
-"O	c black",
-"+	c #333333",
-"@	c #111111",
-"#	c #FF6666",
-"$	c #FEFE00",
-"%	c #DDDDDD",
-"&	c #BBBBBB",
-/* pixels */
+" 	c None",
+".	c #999999",
+"+	c #000000",
+"@	c #333333",
+"#	c #111111",
+"$	c #FF6666",
+"%	c #FEFE00",
+"&	c #DDDDDD",
+"*	c #BBBBBB",
 "                                ",
 "                                ",
 "                                ",
@@ -21,27 +18,27 @@ static char * icl8_11432_xpm[] = {
 "                                ",
 "                                ",
 "                                ",
-"           oOoOo                ",
-"          O     Oo              ",
-"         o        O             ",
-"         O        o             ",
-"       +++++       O            ",
-"       +@@@+       o            ",
-"      +@@O@@+       O           ",
-"    ++@OOO@O@++     o  #       $",
-"   +@@OOOOOOO@@+    O      $  # ",
-"  +@OOOOOOOOOOO@+    o  $       ",
-"  +@OOOOOO%&oOO@+    O   #  $   ",
-" +@OOOOOOOO%&oOO@+    o         ",
-" +@OOOOOOOOOOOOO@+     OoO # $ #",
-" +@OOOOOOOOO%&oO@+              ",
-" +@OOOOOOOOO%&oO@+      $   $   ",
-" +@OOOOOOOOO%&oO@+        #  #  ",
-" +@OOOOOOOO%&oOO@+     #        ",
-" +@OOOOOOOO%&oOO@+    $   $    $",
-"  +@OOOOOO%&oOO@+               ",
-"  +@OOOOOOOOOOO@+         #     ",
-"   +@@OOOOOOO@@+                ",
-"    ++@@@@@@@++                 ",
-"      +++++++                   ",
+"           .+.+.                ",
+"          +     +.              ",
+"         .        +             ",
+"         +        .             ",
+"       @@@@@       +            ",
+"       @###@       .            ",
+"      @##+##@       +           ",
+"    @@#+++#+#@@     .  $       %",
+"   @##+++++++##@    +      %  $ ",
+"  @#+++++++++++#@    .  %       ",
+"  @#++++++&*.++#@    +   $  %   ",
+" @#++++++++&*.++#@    .         ",
+" @#+++++++++++++#@     +.+ $ % $",
+" @#+++++++++&*.+#@              ",
+" @#+++++++++&*.+#@      %   %   ",
+" @#+++++++++&*.+#@        $  $  ",
+" @#++++++++&*.++#@     $        ",
+" @#++++++++&*.++#@    %   %    %",
+"  @#++++++&*.++#@               ",
+"  @#+++++++++++#@         $     ",
+"   @##+++++++##@                ",
+"    @@#######@@                 ",
+"      @@@@@@@                   ",
 "                                "};
